components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -151,10 +174,14 @@ export function CreateWorkspacePage() {
                 <div className="w-full flex justify-end mt-6 space-x-2 px-6">
                     <Button
                         onClick={onClickCreate}
-                        loading={createWorkspaceMutation.isLoading}
+                        loading={createWorkspaceMutation.isLoading || isLoading}
                         disabled={!repo || repo.length === 0 || !!errorIde || !!errorWsClass}
                     >
-                        {createWorkspaceMutation.isLoading ? "Creating Workspace ..." : "New Workspace"}
+                        {isLoading
+                            ? "Thinking ..."
```

Comment:
```suggestion
                            ? "Thinking ..."
```
Is there a better word for this? Something like "Preparing ..." which doesn't make it shound like a troll?

Reply:
haha, of course. 😆

Reply:
How about "Loading ..."?